### PR TITLE
Integration test

### PR DESCRIPTION
**Pull request by vshanthe**
## 📝 Description
Added test for account and users 

## ✔️ How to Test
make MODULE=account testint 
make MODULE=users testint 

## 📷 Preview

**If applicable, include a screenshot or code snippet of this change. Otherwise, please remove this section.**

### CODE REVIEW

**Review by lgarber-akamai — APPROVED** [review]
Looks great and tests are passing on my end. Nice work!

File: tests/integration/lke/test_clusters.py [hunk 1]
```diff
@@ -457,44 +457,6 @@ def test_node_pool(test_lke_cluster):
...
-        "labels.key",
-        "labels.value",
-        "tags",
-        "taints",
-        "type",
-    ]
-
-    assert_headers_in_lines(headers, lines)
-
-
```
Comment (zliang-akamai) [inline tests/integration/lke/test_clusters.py:497]: Is this an intentional deletion?

Comment (vshanthe) [reply tests/integration/lke/test_clusters.py:497]: same test already exits so

Comment (zliang-akamai) [reply tests/integration/lke/test_clusters.py:497]: Where may I find them? 👀

**Review by zliang-akamai — APPROVED** [review]
LGTM!

**Pull request merged**